ui/src/components/Users/SearchResult/index.tsx

```diff
@@ -52,11 +51,41 @@ const SearchResult: React.FC<Props> = (props): React.ReactElement => (
                 {props.user.firstName}. {props.user?.lastName}
             </span>
             <div className="relative z-20 col-span-4 flex h-full items-center font-light text-grey-600 transition-colors duration-500 dark:text-grey-100 lg:col-span-3">
-                {/**
-                 * @TODO - will display current author affiliation here
-                 * GET /users endpoint doesn't return affiliations atm
-                 * This will be handled in OCT-385
-                 */}
+                {props.user.employment
```

Comment:
Could we sensibly add a test for this? or would we need to build too much

Reply:
We currently don't have any tests file for this component. If we create one, we should test the entire component. Should I do it into this ticket?

Reply:
either here or we create a ticket and see if we can add that to the sprint, discuss at standup